### ISSUE TITLE
DefaultModal didn´t work in Chrome

### ISSUE THREAD

**Issue by AngelaIp** [ArasLabs/custom-modal-dialog#1]
Hi Eli,
I  have tested this Project under SP9 and it works fine for IE and Firefox. But for Chrome Version 61.0.3163.100 the Form does not open and I get the following error message:

Technical Message | aras_object: "undefined" : "undefined

The following line makes trouble in Chrome:
`var res = top.aras.modalDialogHelper.show("DefaultModal", window, param, options, "ShowFormAsADialog.html");`

If I replace "DefaultModal" with "DefaultPopup" the Form opens. The "Dialog aborted" alert is immediately thrown, but this is not a big problem.

What is the difference between these two Dialogs and is there a Chrome fix avaible?

Best regards!
Angela

**Comment by EliJDonahue** [comment]
Hi Angela,

I believe `top.aras.modalDialogHelper` is deprecated in 11 SP9 due to changes in the newer tabbed client. 

Give this a shot instead:

```javascript
var topWnd = aras.getMostTopWindowWithAras(window);
topWnd = topWnd.main || topWnd;

var params = {
    title: 'Sample Popup Dialog', 
    formId: 'id',                       // put your form's id here
    aras: aras,
    dialogWidth: width,                 // set width int
    dialogHeight: height,               // set height int
    content: 'ShowFormAsADialog.html'
};

var callback = function(res) {
    //Perform logic using dialog result here
}

topWnd.ArasModules.Dialog.show("iframe", params).promise.then(callback);
```

Eli

**Comment by AngelaIp** [comment]
Hi Eli,

I have made a short test and this Method seems to be a suitable and working replacement!
In the meanwhile, I was already able to realize my custom dialog with DefaultPopup and some different approach to handle the Form values. Maybe I can provide this project next week.

Angela

**Comment by EliJDonahue** [comment]
That sounds great, Angela! I’ll leave this issue open for now if you want to submit a pull request with your fix. 
I don’t want to steal your chance for a [Hacktoberfest](http://community.aras.com/join-aras-labs-hacktoberfest-2017/) pull request! 🙂

If the issue is still unresolved at the end of the month, I or someone from Aras Labs will push a fix.

**Comment by EliJDonahue** [comment]
Closing the issue, as it was resolved by pull request #2.
